src/EnergyPlus/HeatPumpWaterToWaterSimple.cc

```diff
@@ -654,6 +761,488 @@ namespace HeatPumpWaterToWaterSimple {
+						ReportSizingManager::ReportSizingOutput( "HeatPump:WaterToWater:EquationFit:Cooling",GSHP( GSHPNum ).Name, "Design Size Cooling Power Consumption [W]", tmpPowerDraw, "User-Specified Cooling Power Consumption [W]", nomPowerDrawUser );
+					} else {
+						ReportSizingManager::ReportSizingOutput( "HeatPump:WaterToWater:EquationFit:Cooling",GSHP( GSHPNum ).Name,"User-Specified Cooling Power Consumption [W]", nomPowerDrawUser );
+					}
+					if ( DataGlobals::DisplayExtraWarnings ){
+						if ( ( std::abs( tmpPowerDraw - nomPowerDrawUser ) / nomPowerDrawUser ) > DataSizing::AutoVsHardSizingThreshold ) {
+							ShowMessage( "sizeCoolingWaterToWaterHP: Potential issue with equipment sizing for " + GSHP( GSHPNum ).Name );
+							ShowContinueError( "User-Specified Cooling Power Consumption of " + General::RoundSigDigits( nomPowerDrawUser, 2 ) + " [W]" );
+							ShowContinueError( "differs from Design Size Cooling Power Consumption of " + General::RoundSigDigits( tmpPowerDraw, 2 ) + " [W]" );
+							ShowContinueError( "This may, or may not, indicate mismatched component sizes." );
+							ShowContinueError( "Verify that the value entered is intended and is consistent with other components." );
+						}
+					}
+				}
+				tmpPowerDraw = nomPowerDrawUser;
+				GSHP( GSHPNum ).refCOP = tmpCoolingCap / tmpPowerDraw;
+			}
+		}
+
+		PlantUtilities::RegisterPlantCompDesignFlow( GSHP( GSHPNum ).LoadSideInletNodeNum, tmpLoadSideVolFlowRate );
+		// only register half of the source side flow because we expect a companion heat pump to also register a flow and we don't want to double count
+		PlantUtilities::RegisterPlantCompDesignFlow( GSHP( GSHPNum ).SourceSideInletNodeNum, tmpSourceSideVolFlowRate * 0.5 );
+
+		if ( DataPlant::PlantFinalSizesOkayToReport ) {
+			//create predefined report
+			OutputReportPredefined::PreDefTableEntry( OutputReportPredefined::pdchMechType, GSHP( GSHPNum ).Name, "HeatPump:WaterToWater:EquationFit:Cooling" );
+			OutputReportPredefined::PreDefTableEntry( OutputReportPredefined::pdchMechNomEff, GSHP( GSHPNum ).Name, GSHP( GSHPNum ).refCOP );
+			OutputReportPredefined::PreDefTableEntry( OutputReportPredefined::pdchMechNomCap, GSHP( GSHPNum ).Name, GSHP( GSHPNum ).RatedPowerCool );
+		}
+
+		if ( errorsFound ) {
+			ShowFatalError( "Preceding sizing errors cause program termination" );
+		}
+	}
+
+	void
+	sizeHeatingWaterToWaterHP( int const GSHPNum ) // GSHP Number
+	{
+		//do sizing related calculations and reporting for heating heat pumps
+		bool errorsFound( false );
+		static std::string const RoutineName( "sizeHeatingWaterToWaterHP" );
+		Real64 tmpLoadSideVolFlowRate = GSHP( GSHPNum ).RatedLoadVolFlowHeat;
+		Real64 tmpSourceSideVolFlowRate = GSHP( GSHPNum ).RatedSourceVolFlowHeat;
+		Real64 tmpHeatingCap = GSHP( GSHPNum ).RatedCapHeat;
+		Real64 tmpPowerDraw= GSHP( GSHPNum ).RatedPowerHeat;
+
+		//if companion cooling coil known, update info from that
+		if ( GSHP( GSHPNum ).companionIdentified ) {
```

Comment:
This is interesting.  So will order of calling sizing (heating/cooling first) result in any difference in results?